shacl12-core/index.html

```diff
@@ -6413,6 +6411,98 @@ <h4>sh:property</h4>
 						although it is reached via different paths in the <a>shapes graph</a>.
 					</p>
 				</section>
+
+				<section id="SomeValueConstraintComponent">
+					<h4>sh:someValue</h4>
+					<p>
+						<code>sh:someValue</code> specifies the condition that at least one <a>value node</a> conforms to the given <a>shape</a>.
+					</p>
+					<p>
+						<span class="component-class">Constraint Component IRI</span>: <code>sh:SomeValueConstraintComponent</code>
+					</p>
+
+					<div class="parameters">Parameters:</div>
+					<table class="term-table">
+						<thead>
+							<tr>
+								<th>Property</th>
+								<th>Summary and Syntax Rules</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr>
+								<td><code>sh:someValue</code></td>
+								<td>
+									The <a>shape</a> that at least one of the <a>value nodes</a> needs to conform to.
+									<span data-syntax-rule="someValue-shape">The values of <code>sh:someValue</code> in a shape must be <a>well-formed</a> <a>shapes</a>.</span>
+								</td>
+							</tr>
+						</tbody>
+					</table>
+					<div class="def def-text">
+						<div class="def-header">TEXTUAL DEFINITION</div>
+						<div class="def-text-body" data-validator="Node">
+							Let <code>$someValue</code> be a <a>value</a> of <code>sh:someValue</code>.
+							A <a>failure</a> MUST be produced if the <a>conformance checking</a> of any <a>value node</a> against
+							<code>$someValue</code> produces a <a>failure</a>, unless at least one of the <a>value nodes</a> conforms to <code>$someValue</code>.
```

Comment:
Is this right?  If the existence condition by some node is satisfied, a **failure** is overridden?  Is this conflating failure and violation?

Reply:
@HolgerKnublauch the informative part matches `sh:or`, but there is a mismatch in the textual definition. That should be aligned. Your definition for `sh:someValue` matches the informative parts, and probably also the reality of current implementations. I can confirm that shacl-engine skips any validations for `sh:or` after the first successful validation, unless debug mode is enabled.

Reply:
Recorded as #553 which should be handled independently of this one here.

I will merge this PR into master after the next meeting, unless I hear otherwise.